kubeflow/jupyter/notebook_controller.libsonnet

```diff
@@ -1,32 +1,32 @@
 {
-  local util = import "kubeflow/common/util.libsonnet",
```

Comment:
we use double quote else where.
I think maybe your editor/formatter did this?
Could you revert the change here

Reply:
yea... it's `vim-jsonnet`...  reverted.